### PR TITLE
Get symbols for an exchange

### PR DESCRIPTION
**Pull request by mathu97**
See IEX International symbols: https://iexcloud.io/docs/api/#international-symbols

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,111 +1,118 @@
+### 1.4.2 (Next)
+
+- Your contribution here.
+- [#103](https://github.com/dblock/iex-ruby-client/pull/103): Added support for fetching symbols for an exchange - [@mathu97](https://github.com/mathu97).
+
 ### 1.4.1 (Next)
-* Your contribution here.
```
Comment (mathu97) [inline CHANGELOG.md:2]: 🤦 autoformatting.

**Comment by dblock** [comment]
Thank you!

**Comment by dblock** [comment]
Want to help out with this gem, @mathu97? Make the next release? Email me your rubygems username (dblock at dblock dot org) if you're interested.

**Comment by mathu97** [comment]
> Want to help out with this gem, @mathu97? Make the next release? Email me your rubygems username (dblock at dblock dot org) if you're interested.

sent!

**Pull request merged**